File: src/nomanssave/cS.java
package nomanssave;

class cS implements gD {
   final String filename;
   final int index;
   // $FF: synthetic field
   final cN gt;

   cS(cN var1, String var2) {
      this.gt = var1;
      this.filename = var2;
      this.index = cN.a(var1).size() + 1;
   }

   public String K() {
      return this.filename;
   }

   public String toString() {
      return "Unknown " + this.index;
   }
}

File: NMSSaveEditor/nomanssave/mixed/cS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{
public class cS : gD {
   public string filename;
   public int index;
   cN gt;

   public cS(cN var1, string var2) {
      this.gt = var1;
      this.filename = var2;
      this.index = cN.a(var1).Count + 1;
   }

   public string K() {
      return this.filename;
   }

   public string toString() {
      return "Unknown " + this.index;
   }
}

}
